miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniappSdkInitializer.kt

```diff
@@ -58,16 +65,28 @@ class MiniappSdkInitializer : ContentProvider() {
 
         MiniApp.init(
             context = context,
-            baseUrl = manifestConfig.baseUrl(),
-            rasAppId = manifestConfig.rasAppId(),
-            subscriptionKey = manifestConfig.subscriptionKey(),
-            hostAppVersionId = manifestConfig.hostAppVersion(),
-            isTestMode = manifestConfig.isTestMode()
+            miniAppSdkConfig = MiniAppSdkConfig(
+                baseUrl = manifestConfig.baseUrl(),
+                rasAppId = manifestConfig.rasAppId(),
+                subscriptionKey = manifestConfig.subscriptionKey(),
+                hostAppVersionId = manifestConfig.hostAppVersion(),
+                hostAppInfo = provideHostAppInfo(context, manifestConfig),
+                isTestMode = manifestConfig.isTestMode()
+            )
         )
 
         return true
     }
 
+    @VisibleForTesting
+    internal fun provideHostAppInfo(context: Context, manifestConfig: AppManifestConfig): String =
+        if (manifestConfig.hostAppInfo().isEmpty()) {
```

Comment:
Please check my other comment. 
It needs to be explicitly set by consumer app as it is an entity for the host app's identification (and can involve more params like version etc.)

Reply:
Okay. I will not add the default string here.